### PR TITLE
Fix for Grepper results on Google and Hackerrank badge icons.

### PR DESCRIPTION
**Pull request by mcAlex42**

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -9142,6 +9142,16 @@ IGNORE IMAGE ANALYSIS
...
 
+google.com
+
+INVERT
+.tays_show_more_answer_button_hide
+.tays_show_more_answer_button_show
+#tays_add_more_answers_button_hide
+#tays_add_more_answers_button
+
+================================
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:9153]: Add these fixes instead at `www.google.*`

**Comment by Gusted** [comment]
Hi @mcAlex42,

The changes look good, but could you link the google page where I can observe the changes made to google

**Comment by mcAlex42** [comment]
The changes on google apply to results from the Grepper web extension. I have attached screenshots: 

<p align="center">
Old
Old
</p>

**Comment by Gusted** [comment]
image
Not able to verify for now... I will merge it.

**Review by Gusted — APPROVED** [review]

**Comment by mcAlex42** [comment]
Ah, I think there might have been a misunderstanding. The code snippets you saw on my screenshots are from the Grepper extension. They won't show up unless you have the extension installed.

**Pull request merged**